### PR TITLE
Fix logic for excluding relations

### PR DESCRIPTION
**Pull request by mdmay74**
Fix logic to handle dbRelationDontEnforce bit even if other bits are set.

Great project, I'm finding it really useful. 
This is my first contribution through GitHub, so I hope I've done the right thing. Any questions, please don't hesitate to ask :)
Matt.

### CODE REVIEW

**Comment by timabell** [comment]
> Great project, I'm finding it really useful.

I'm glad it's helpful :-)
It's the work of many generous people.

> This is my first contribution through GitHub, so I hope I've done the right thing.

Yep this is great, thanks for sharing the improvements.

I'll make a couple of comments on your patch, but I'll leave leave it open to discussion with other more active contributors.

File: MSAccess-VCS/VCS_ImportExport.bas [hunk 1]
```diff
@@ -279,11 +279,13 @@ Err_TablePFSNotFound:
                 For Each aRelation In CurrentDb.Relations
                         ' Exclude relations from system tables and inherited (linked) relations
                         ' Skip if dbRelationDontEnforce property is not set. The relationship is already in the table xml file. - sean
+			' Fix logic to handle dbRelationDontEnforce bit even if other bits are set. - mdmay74
```
Comment (timabell) [inline MSAccess-VCS/VCS_ImportExport.bas:282]: No need to add this comment as this information is already available in the git history. Please remove it from your patch.

File: MSAccess-VCS/VCS_ImportExport.bas [hunk 2]
```diff
@@ -279,11 +279,13 @@ Err_TablePFSNotFound:
...
                         ' Exclude relations from system tables and inherited (linked) relations
                         ' Skip if dbRelationDontEnforce property is not set. The relationship is already in the table xml file. - sean
+			' Fix logic to handle dbRelationDontEnforce bit even if other bits are set. - mdmay74
                         If Not (aRelation.Name = "MSysNavPaneGroupsMSysNavPaneGroupToObjects" _
                                         Or aRelation.Name = "MSysNavPaneGroupCategoriesMSysNavPaneGroups" _
                                         Or (aRelation.Attributes And DAO.RelationAttributeEnum.dbRelationInherited) = _
                                         DAO.RelationAttributeEnum.dbRelationInherited) _
-                                        And (aRelation.Attributes = DAO.RelationAttributeEnum.dbRelationDontEnforce) Then
+                           And ((aRelation.Attributes And DAO.RelationAttributeEnum.dbRelationDontEnforce) = _
+			  		DAO.RelationAttributeEnum.dbRelationDontEnforce) Then
```
Comment (timabell) [inline MSAccess-VCS/VCS_ImportExport.bas:288]: Is the difference in indentation here intentional or are you using a different tab/space setting to the rest of the file?

**Comment by timabell** [comment]
Oh and I see you're just back into software, so do reach out if I can help in any way. I never left software :-) tim@timwise.co.uk

**Comment by mdmay74** [comment]
Okay, what is the best way to remove the comment from my patch? Just remove
the line from my GitHub repo and make a new pull request? (Or would that
effectively register as a new fix?) I haven't really use git yet, but I am
starting to learn. I made a unified diff patch in bash before forking the
project on GitHub and uploading the new file from there...

The difference in indentation is deliberate. I felt that:
If Not (a
        Or b
        Or c)
   And d Then

better reflected what was happening. (At first glance I thought it was
doing something else, until I noticed the closing parenthesis after
dbRelationInherited, and re-read the comments.)

Yes, been out of paid work for a while after my wife and I left the city
and built a home in a rural area. Thanks for the offer, I really feel
strongly about the Open Source environment of collaboration :-)
Funnily enough, when I first saw your name, I thought of a Bell-Ringing
simulator that I was shown when I was learning handbells before I left the
UK - but that has/had only one "l" :)

Cheers,
Matt.

**Pull request merged**